src/OrchardCore.Modules/OrchardCore.Users/Drivers/UserDisplayDriver.cs

```diff
@@ -79,11 +93,26 @@ public override async Task<IDisplayResult> UpdateAsync(User user, UpdateEditorCo
             model.UserName = model.UserName?.Trim();
             model.Email = model.Email?.Trim();
 
+            var usersOfAdminRole = await _userManager.GetUsersInRoleAsync("Administrator");
             if (!model.IsEnabled && user.IsEnabled)
             {
-                user.IsEnabled = model.IsEnabled;
-                var userContext = new UserContext(user);
-                await Handlers.InvokeAsync((handler, context) => handler.DisabledAsync(userContext), userContext, _logger);
+                if (user.UserName != httpContext.User.Identity.Name)
+                {
+                    if(usersOfAdminRole.Count == 1 && usersOfAdminRole.First().UserName == user.UserName)
+                    {
+                        _notifier.Warning(H["Cannot disable unique administrator."]);
```

Comment:
Cannot disable the super administrator.

Reply:
Well technically it's not a super administrator but a simple administrator. I get what you mean though.

Reply:
Still I feel "unique" is annoying ;)